snapd-glib/snapd-notices-monitor.c

```diff
@@ -17,6 +17,28 @@
 
 #include "snapd-notices-monitor.h"
 
+/**
+ * SECTION: snapd-notices-monitor
+ * @short_description: Allows to receive events from snapd.
+ * @include: snapd-glib/snapd-glib.h
+ *
+ * #SnapdNoticesMonitor allows to receive in real time events from
+ * snapd, like status changes in an ongoin refresh, inhibited refreshes
+ * due to the snap being active, or inhibited launches due to an ongoin
+ * refresh.
+ */
+
+/**
+ * SnapdNoticesMonitor:
+ *
+ * #SnapdNoticesMonitor allows to receive in real time events from
+ * snapd, like status changes in an ongoin refresh, inhibited refreshes
+ * due to the snap being active, or inhibited launches due to an ongoin
```

Comment:
ongoin -> ongoing

Reply:
(twice in this paragraph)

Reply:
Agh! The spell checker was disabled... Fixed. Sorry O:-)